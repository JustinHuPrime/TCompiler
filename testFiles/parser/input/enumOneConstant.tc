module foo;

enum bar {
  A
};