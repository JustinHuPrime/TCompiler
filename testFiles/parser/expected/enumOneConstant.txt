testFiles/parser/input/enumOneConstant.tc (code):
FILE(1, 1, STAB(ENTRY(bar, ENUM(testFiles/parser/input/enumOneConstant.tc, 3, 1, CONSTANT(A, 0)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), ENUMDECL(3, 1, ID(3, 6, bar, REFERENCES(testFiles/parser/input/enumOneConstant.tc, 3, 1)), ID(4, 3, A, REFERENCES(testFiles/parser/input/enumOneConstant.tc, 4, 3)), (null)))
